docs.feldera.com/docs/operations/metrics.md

```diff
@@ -0,0 +1,161 @@
+<!-- This file is automatically generated.  Do not edit!
+
+To regenerate this file, start a Feldera pipeline (any one will do)
+and obtain Prometheus metrics for it with, for example, `fda metrics
+--format=prometheus` or `curl https://server/v0/metrics`, then run
+`metrics.py` from the same directory as this, like so:
+
+metrics.py < metrics.txt > metrics.md
+-->
+
+# Metrics
```

Comment:
The title is a bit terse. "Metrics collected for executing pipelines"?
(There could be metrics for other services too)